canfund-rs/src/manager/record.rs

```diff
@@ -11,36 +11,40 @@ pub struct CanisterRecord {
     /// The canister cycles balance record when it was last funded.
     previous_cycles: Option<CyclesBalance>,
     /// The cycles consumption history of the canister.
-    consumtion_history: ConsumptionHistory,
+    consumption_history: ConsumptionHistory,
     /// The cumulative total of cycles deposited to the canister.
     deposited_cycles: Option<CyclesBalance>,
     /// The method to fetch the canister cycles balance.
     cycles_fetcher: Arc<dyn FetchCyclesBalance>,
     /// Optional fund strategy for the canister which overrides the global strategy.
     strategy: Option<FundStrategy>,
+    /// Optional minting strategy for the canister which overrides the global strategy.
+    obtain_cycles_config: Option<ObtainCyclesOptions>,
```

Comment:
this config on the manager itself is called `ObtainCyclesOptions` right? any reason to not name this the same instead of `obtain_cycles_config`?

Reply:
IMO "options" is misleading in the context of canfund, as it's not about choosing from existing options, but rather about configuring the strategy with custom inputs. I could have renamed the original structure too, but that would introduce even more breaking changes. But it could be bundled with the set of breaking changes in the v1 after all. WDYT?